inbrain-surveys.podspec

```diff
@@ -20,6 +20,6 @@ Pod::Spec.new do |s|
   s.source_files = "ios/*.{h,m,swift}"
 
   s.dependency "React"
-  s.dependency 'InBrainSurveys', '~> 2.4.3'
+  s.dependency 'InBrainSurveys'
```

Comment:
the version range is missing, should be:
 s.dependency 'InBrainSurveys', '~> 2.5.0'

Reply:
Done